src/backend/catalog/storage.c

```diff
@@ -58,11 +58,21 @@ typedef struct PendingRelDelete
 	RelFileNodePendingDelete relnode;		/* relation that may need to be deleted */
 	bool		atCommit;		/* T=delete at commit; F=delete at abort */
 	int			nestLevel;		/* xact nesting level of request */
+	dsa_pointer shmem_ptr;		/* ptr to shared pending delete list node */
```

Comment:
Other fields of the structure are in CamelCase. For code style consistency I suggest to use CamelCase here as well:
```suggestion
	dsa_pointer shmemPtr		/* ptr to shared pending delete list node */
```

Reply:
Fixed